CyanEmu/Scripts/CyanEmuPickupHelper.cs

```diff
@@ -10,7 +10,7 @@ public class CyanEmuPickupHelper : MonoBehaviour, ICyanEmuInteractable
         private const float INITIAL_PICKUP_DURATION_ = 0.5f;
         private const float MAX_PICKUP_DISTANCE_ = 0.25f;
         private static Quaternion GRIP_OFFSET_ROTATION_ = Quaternion.Euler(0, 45, 0);
```

Comment:
I just checked the rotations in VRChat. Your update to Gun offset is correct, but it looks like Grip also needs to be modified. VRChat reports 35 degrees relative to the hand tracker. Other than that, everything else is good to be merged in. 

```
Vector3 gripRelPos = Vector3.zero;
Quaternion gripRelRot = Quaternion.identity;
if (gripPickup.IsHeld)
{
    Transform gripPickupTransform = gripPickup.transform;
    Transform prevGripPar = gripPickupTransform.parent;
    gripPickupTransform.SetParent(rightHandTransform, true);
    gripRelPos = gripPickupTransform.localPosition;
    gripRelRot = gripPickupTransform.localRotation;
    gripPickupTransform.SetParent(prevGripPar, true);
}
```
![Screenshot 2021-07-28 154932](https://user-images.githubusercontent.com/50863419/127386530-1b592be4-7378-4a67-bf07-b5291d0eb76c.png)